slsReceiverSoftware/include/HDF5FileStatic.h

```diff
@@ -47,19 +47,18 @@ class HDF5FileStatic: public virtual slsDetectorDefs {
 	 * @param unitindex unit index
```

Comment:
I would correct the comment to say it is the sub frame index and not frame number index. I would also remove frindexenable from the comment. I would use "subfIndex" rather than fnum as a variable name.

Reply:
ok, done